Oleksandr.Svidovyi/Task_6/src/main/java/com/hillel/task_6/controller/AccountController.java

```diff
@@ -0,0 +1,54 @@
+package com.hillel.task_6.controller;
+
+import com.hillel.task_6.model.Account;
+import com.hillel.task_6.service.ClientService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/clients/{id}/account")
+public class AccountController {
+    @Autowired
+    private ClientService clientService;
+
+    @RequestMapping(method = RequestMethod.GET, value = "/list")
+    public ModelAndView getAccounts(@PathVariable int id) {
+        List<Account> accounts = clientService.getClient(id).getAccounts();
```

Comment:
Since we fetch clientId from URL wich is formed from concrete client, I asume client already exists.

Method clientService.getAccountsByClientId(int clientId) added.